backend/metering_billing/views/model_views.py

```diff
@@ -426,6 +427,19 @@ class EventViewSet(
     def get_queryset(self):
         now = now_utc()
         organization = self.request.organization
+        distinct = self.request.query_params.get("distinct")
```

Comment:
This is bad practice, we are essentially hijacking the functionality of a helper method (get queryset is used by every action in a viewset pretty much). This method should do exactly what it's supposed to, which is defined here:

https://www.django-rest-framework.org/api-guide/generic-views/#get_querysetself

only as the base for object specific methods.

Reply:
Nice catch here. I will try to refactor this to not override the base helper since it can have unintended consequences